lib/src/models/comment.dart

```diff
@@ -32,7 +32,7 @@ class CommentListModel with _$CommentListModel {
                   possibleChildren: json['comments'] as List<dynamic>,
                 ))
             .toList(),
-        nextPage: null,
+        nextPage: json['next_page'] as String?,
```

Comment:
@olorin99, I just checked the [lemmy comment api](https://lemmy.readme.io/reference/get_comment-list), and I don't think it actually has a `next_page` key.

Reply:
Oh, never mind, I saw your comment about it. That's actually pretty smart; I was running into the same thing too.